contracts/smart-contract-wallet/base/FallbackManager.sol

```diff
@@ -50,4 +50,6 @@ contract FallbackManager is SelfAuthorized {
             return(0, returndatasize())
         }
     }
+
+    uint256[24] private __gap;
```

Comment:
shouldn't we name this with different variable name rather than __gap which is same in ModuleManager too

Reply:
I think it doesn't matter since it's private.

Reply:
If we want specific names for readability we can name __moduleManagerGap and __fallbackManagerGap